python/cudf/cudf/tests/test_gcs.py

```diff
@@ -0,0 +1,15 @@
+import pandas as pd
+import pytest
+
+import cudf
+from cudf.tests.utils import assert_eq
+
+gcsfs = pytest.importorskip("gcsfs")
+
+
+def test_gcs_public():
```

Comment:
This is slowish (10-15 seconds). (The data itself isn't too bit ~1700 rows). 
The other alternative might be similar to what pandas does where we create a `gcsfs` object but override the open method to open a `BytesIO` object instead. This just tests if adding `gcs://` to the path resolves everything properly but invokes reading python bytes instead. [Example](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/io/test_gcs.py#L30)

Edit: Just found the gcsfs test utils which might also be a good [alternative](https://github.com/dask/gcsfs/blob/master/gcsfs/tests/utils.py#L190)

Reply:
I'm fine with either approach, I don't think we should hit a GCS resource not owned by us for CI :).

Reply:
Done